manual/num/rotagrid.tex

```diff
@@ -15,22 +15,38 @@ \subsubsection{~Rotated grids} \label{sub:num_space_rotagrid}
 grid allows a much more evenly spaced lat-lon mesh in the NAEW domain than the
 standard lat-lon grid in the same area. 
 
-In \ws\, the rotated grid is implemented with minimum changes to the original
+In \ws, the rotated grid is implemented with minimum changes to the original
 lat-lon grid. In fact, the rotated grid is treated just like the standard 
 lat-lon grid inside the model. To set up and run a rotated grid model configuration,
 users should choose the regular lat-lon grid along with the {\code RTD} switch.
-The rotated pole position is set using the {\code PLAT} and {\code PLON} variables in the
-{\bf ww3\_grid.inp} namelist {\code ROTD}. Model input files, like wind, current and ice files 
-should be mapped on to the rotated grid. For convenience of nesting in standard 
-lat-lon grid frameworks, boundary conditions provided to and output from the 
+The rotated pole position is set using the {\code PLAT} and {\code PLON} variables
+under the namelist {\code ROTD} in the input file {\bf ww3\_grid.inp}
+(see Sect.~\ref{sec:config011}). If the pole is set as {\code PLAT = 90.0},
+{\code PLON = -180.0}, the grid is treated as a standard lat-lon system.
+
+Model input files, like wind, current and ice files
+should be mapped on to the rotated grid. For convenience of nesting in standard
+lat-lon grid frameworks, boundary conditions data provided to and output from the
 rotated grid use spectra referenced to a standard grid north and standard lat-lon
-grid points values, which are converted into rotated grid lat-lon inside \ws\.
-The list of 2D spectral output locations in {\bf ww3\_shel.inp} are also specified in 
-standard lat-lon. 
+grid points values, which are converted into rotated grid lat-lon inside \ws.
+The list of 2D spectral output locations in {\bf ww3\_shel.inp} or
+{\bf ww3\_shel.nml} are also specified in standard lat-lon. When the boundary
```

Comment:
This section might need some re-phrasing to clarify how the boundary conditions are set up? Something along the lines of..

When nesting from a standard grid to a rotated grid model, both the outer and inner 
grids should be built with the {\code RTD} switch set when compiling executables for 
the models. When nesting from a rotated grid to a standard grid model, the inner 
(standard) grid model does not require to be built with {\code RTD}.

Output of spectra at boundary points to one-way nested inner grids are transferred
as described in Appendix~\ref{app:nest}. Output b.c.\ are defined in the input
file (see {\bf ww3\_grid.inp}, Sect.~\ref{sec:config011}) as a sequence of straight
lines given in coordinates of the inner grid. If the inner grid is rotated, it's
pole position must be defined as the values of the array elements
{\code BPLAT(\sl{n})} and {\code BPLON(\sl{n})} under namelist {\code ROTB}
for the outer grid. The array index {\sl{n}} is the index ({\sl{1:9}}) of the boundary 
conditions file {\file nest{\sl{n}}.ww3}.

Reply:
This is a good clarification. I consider the manual  the most valuable component of the WW3 distribution.